Examples/Physics_applications/spacecraft_charging/inputs_test_rz_spacecraft_charging_picmi.py

```diff
@@ -71,12 +71,25 @@ def save_normalized_vacuum_Efields(
         self,
     ):
         warpx = sim.extension.warpx
-
+        amrex = sim.externsion.amrex
```

Comment:
CI failure caused by this typo:
```suggestion
        amrex = sim.extension.amrex
```